pkgs/development/libraries/glib/default.nix

```
@@ -208,6 +213,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   postInstall = ''
     moveToOutput "share/glib-2.0" "$dev"
+    # except gdb scripts
+    mkdir -p $out/share/glib-2.0
+    mv $dev/share/glib-2.0/gdb $out/share/glib-2.0
```

Does not the following work?

```suggestion
moveToOutput "share/glib-2.0/gdb" "$out"
```

---

changed